src/com/workspace/utill/LogWriter.java

```diff
@@ -1,22 +1,22 @@
 package com.workspace.utill;
-
+import com.workspace.io.*;
 public class LogWriter {
+    private ConsoleWriter consoleWriter = null;
+    private FileWriter fileWriter = null;
+    public LogWriter(){
```

Comment:
why public? I don't see why we should make this class available to other modules/packages.